app/routes/index.js

```diff
@@ -2,8 +2,25 @@ const express = require('express');
 const router = express.Router();
 
 /* GET home page. */
+/* router.get('/', function(req, res, next) {
+    res.render('index', {title:'OG-YOGA', error: ""});
+}); */
+
+/* GET login page. */
 router.get('/', function(req, res, next) {
-    res.render('index', {title:'OG-YOGA'});
+    res.render('login', {title: 'Login'});
+});
+
+router.post('/login', async function(req, res, next) {
+    if(req.body.secret_key !== process.env.TOKEN_SECRET){
+        const error = "wrong secret key";
+        console.log(error);
```

Comment:
What I mean by sending it to user is to display the error message at the screen. This just logs the error in the console. you can re-render back to the login ejs file and pass another variable like the following syntax: 
```
res.render('login', {title: 'Login', data: error});
```